api/app/routers/threat.py

```diff
@@ -57,23 +57,41 @@ def create_threat(
     )
     persistence.create_threat(db, threat)
 
-    topic = threat.topic
-    if topic and topic.hint_for_action:
-        now = datetime.now()
-        dependency = persistence.get_dependency_from_service_id_and_tag_id(
-            db, service.service_id, tag.tag_id
-        )
-        ticket = models.Ticket(
-            threat_id=str(threat.threat_id),
-            created_at=now,
-            updated_at=now,
-            ssvc_deployer_priority=ssvc.calculate_ssvc_deployer_priority(threat, dependency),
-        )
-        persistence.create_ticket(db, ticket)
-
-        if alert := create_alert_from_ticket_if_meet_threshold(ticket):
-            persistence.create_alert(db, alert)
-            send_alert_to_pteam(alert)
+    actions = persistence.get_actions_by_topic_id(db, data.topic_id)
+    now = datetime.now()
+    action_tag_names_set: set = set()
+
+    if actions:
+        for action in actions:
+            action_tag_names = action.ext.get("tags")
+            if action_tag_names is None:
+                continue
+            for action_tag_name in action_tag_names:
+                action_tag_names_set.add(action_tag_name)
```

Comment:
for ループしないで、set 変換して追加でもいいかもです
https://note.nkmk.me/python-set/#setlisttuple-set-list-tuple
```suggestion
            action_tag_names |= set(action_tag_names)
```